Apps/Src/Tasks.c

```diff
@@ -69,12 +61,12 @@ OS_SEM ArrayConnectionChange_Sem4;
 
 // Needs to get initialized somewhere, not currently initialized
 fault_bitmap_t FaultBitmap;
-os_error_loc_t OSErrLocBitmap = OS_NONE_LOC;
-tritium_error_code_t TritiumErrorBitmap = T_NONE;
+os_error_loc_t OSErrLocBitmap;
+tritium_error_code_t TritiumErrorBitmap;
```

Comment:
Why did you decide to leave these uninitialized?

Reply:
They are initialized in the sysInit function, which should be called in main at the start of our code